src/openrct2/peep/Peep.h

```diff
@@ -249,7 +251,7 @@ enum PEEP_USING_BIN_SUB_STATE
     PEEP_USING_BIN_GOING_BACK,
 };
 
-enum PEEP_ACTION_EVENTS
+enum PEEP_ACTION_EVENTS : uint8_t
```

Comment:
I feel like the name is slightly misleading, they aren't exactly events and the type can only hold one value at a time. Maybe we could rename it to something like PEEP_ACTION_TYPE for consistency.

Reply:
Renamed to PeepActionType in 60396c665861f8541b8d12a88414b538e209cb2f.